integration-tests/opentelemetry-reactive/src/main/java/io/quarkus/it/opentelemetry/reactive/ReactiveResource.java

```diff
@@ -12,10 +12,12 @@
 
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.instrumentation.annotations.WithSpan;
 import io.smallrye.mutiny.Uni;
 
 @Path("/reactive")
 public class ReactiveResource {
+    public static final int SECONDS_DELAY = 1;
```

Comment:
Too long—When I do such tests, I only delay them by 10 or 100ms. The delay itself introduces the I/O / thread switch you want. The duration just makes the test longer.

Reply:
yes... I decreased it already... Will check how brittle is the setup